meta-ros1-melodic/recipes-bbappends/mavros/mavros_0.33.3-1.bbappend

```diff
@@ -0,0 +1,28 @@
+# geographiclib-tools is just a package from geographiclib recipe it should be in
+# RDEPENDS on DEPENDS
+ROS_BUILD_DEPENDS_remove = "${ROS_UNRESOLVED_PLATFORM_PKG_geographiclib-tools}"
+ROS_EXPORT_DEPENDS_remove = "${ROS_UNRESOLVED_PLATFORM_PKG_geographiclib-tools}"
+ROS_EXEC_DEPENDS += "geographiclib-tools"
+
+# ERROR: mavros-1.0.0-1-r0 do_package_qa: QA Issue: /opt/ros/melodic/lib/mavros/install_geographiclib_datasets.sh contained in package mavros requires /bin/bash, but no providers found in RDEPENDS_mavros? [file-rdeps]
+ROS_EXEC_DEPENDS += " \
+    bash \
+"
+
+# This is needed only for old webOS OSE, with newer OSE this should respect
```

Comment:
Is webOS OSE v2.2 new enough to drop this?

Reply:
Not to drop completely, but replace with WEBOS_PREFERRED_PROVIDER_FOR_BASH
as I already did for meta-ros2-dashing/recipes-bbappends/marti-common/swri-prefix-tools_3.0.3-1.bbappend but copy pasted the older version of this stanza here, will fix.